src/leddrv.c

```diff
@@ -28,15 +30,20 @@ static void gpio_buf_set(pinctrl_t pinctl, tristate_t state)
 	}
 }
 
+void led_setDriveStrength(int is_20mA)
```

Comment:
variable with a getter name ?!

Reply:
I can't find any more suitable name. 

My first idea was trying to make this .h arch independent, but I wanted to go fast so this was added as a workaround. I'm thinking of redesigning this.